ui-library-add-call-recording/api/startRecording.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace Contoso
+{
+    public static class startRecording
+    {
+        [FunctionName("startRecording")]
+        public static async Task<IActionResult> Run(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = null)] HttpRequest req,
+            ILogger log)
+        {
+            string requestBody = await new StreamReader(req.Body).ReadToEndAsync();

```

Comment:
I am curious if the request body has nothing and we try to read it. Do we need to do some try/catch magic here?

Reply:
Checking as I implement the start/stop functions for real.